src/main/resources/static/css/style.css

```diff
@@ -134,3 +134,16 @@ li {
     font-size: 18px;
     padding: 10px;
 }
+.logout-form {
```

Comment:
this is getting really big - you should devide it in multiple .css files (i.e. one for login/logout, one for forms and one for more general styles) and import the ones that are needed in certain htmls. also: please remember to format so that the css remains readable